source/administration/devops-command-center.rst

```diff
@@ -120,29 +120,33 @@ To view details of active incidents, select **! Incidents** from the plugin menu
 
 Listed incidents include the following information:
 
-- The current commander’s profile picture and username
-- Zero or more checklist items
-- The channels associated with the incident
-- A button to end the incident
+- The current commander’s profile picture and username.
+- Zero or more checklist items.
+- The channels associated with the incident.
+- A button to end the incident.
 
 If the incident is active, but in a private channel, you won't be able to see the incident channel name or select **End Incident**. However, the commander name and incident checklist are displayed. If you're a participant in an incident channel, you can modify incident details from within that channel.
 
 You can view all incidents in the incident Backstage via **Main Menu > Incidents & Playbooks** and then select an incident to view the following details:
 
 - Incident name
+
 - A link icon to open the corresponding incident channel
+
 - Status (**Ongoing** or **Ended**)
+
```

Comment:
Just out of curiosity: why are adding blank lines here?

Reply:
Formatting error; fixed now! :)